kv/kvserver: TestLeaseRequestBumpsEpoch failed

kv/kvserver.TestLeaseRequestBumpsEpoch [failed](https://mesolite.cluster.engflow.com/invocations/default/5f27edbe-98ef-4fa4-9a35-ae4151c13531?testReportRun=3&testReportShard=31&testReportAttempt=1#targets-Ly9wa2cva3Yva3ZzZXJ2ZXI6a3ZzZXJ2ZXJfdGVzdA==) on release-25.4.0-rc @ [4f4179243a351bc48c00dd9243dc56b8af98e210](https://github.com/cockroachdb/cockroach/commits/4f4179243a351bc48c00dd9243dc56b8af98e210):


```
=== RUN   TestLeaseRequestBumpsEpoch
    test_log_scope.go:171: test logs captured to: outputs.zip/logTestLeaseRequestBumpsEpoch1047195255
    test_log_scope.go:82: use -show-logs to present logs inline
    client_lease_test.go:1806: -- test log scope end --
test logs left over in: outputs.zip/logTestLeaseRequestBumpsEpoch1047195255
--- FAIL: TestLeaseRequestBumpsEpoch (230.76s)
=== RUN   TestLeaseRequestBumpsEpoch/lease-type=LeaseLeader
    test_server_shim.go:561: DRPC is enabled (override by TestingGlobalDRPCOption)
    client_raft_helpers_test.go:543: from store 2; adding store 1 to drop list
    client_raft_helpers_test.go:543: from store 3; adding store 1 to drop list
    client_raft_helpers_test.go:543: from store 1; adding store 2 to drop list
    client_raft_helpers_test.go:543: from store 1; adding store 3 to drop list
    client_raft_helpers_test.go:589: test: waiting for leadership to fail over
    client_raft_helpers_test.go:590: condition failed to evaluate within 3m45s: from client_raft_helpers_test.go:596: no leader yet
--- FAIL: TestLeaseRequestBumpsEpoch/lease-type=LeaseLeader (227.99s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>deadlock=true</code>
 - <code>run=3</code>
 - <code>shard=31</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/kv-triage
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestLeaseRequestBumpsEpoch.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-56381

--- a/pkg/kv/kvserver/client_lease_test.go
+++ b/pkg/kv/kvserver/client_lease_test.go
@@ -1728,6 +1728,9 @@ func TestLeaseRequestBumpsEpoch(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
 
+	// This is to help investigate failures like #157128.
+	testutils.SetVModule(t, "raft=4")
+
 	testutils.RunValues(t, "lease-type", roachpb.TestingAllLeaseTypes(), func(t *testing.T, leaseType roachpb.LeaseType) {
 		ctx := context.Background()
 		st := cluster.MakeTestingClusterSettings()

